### PR TITLE
Added long press and tap gesture to Symbol and Circle features

### PR DESCRIPTION
**Pull request by Archdoog**
- Adds long press and tap gesture to Circle, CircleWithLabel and Symbol
- Adds additional info for gesture context.
- Makes MapLibre internal.

### CODE REVIEW

File: compose/src/main/java/com/maplibre/compose/ramani/MapLibreCallbacks.kt [hunk 1]
```diff
@@ -47,13 +64,32 @@ internal fun MapboxMap.setupEventCallbacks(
...
+}
+
+fun MapboxMap.getFeaturesAtScreenLocation(screenLocation: PointF): List<Feature> {
+    val features = queryRenderedFeatures(screenLocation)
+
+    if (features.isEmpty()) {
+        return emptyList()
+    }
+
+    return features
```
Comment (ianthetechie) [inline compose/src/main/java/com/maplibre/compose/ramani/MapLibreCallbacks.kt:94]: It looks like the net effect is to just return the result of `queryRenderedFeatures` but it creates a fresh empty list in the case that the original list is already empty 🤔 Am I missing something?

File: compose/src/main/java/com/maplibre/compose/ramani/MapLibreCompose.kt [hunk 2]
```diff
@@ -172,6 +177,22 @@ internal class MapApplier(
             zIndexReferenceAnnotationManagerMap[zIndex] = circleManager
         }
 
+        circleManager.addClickListener(OnCircleClickListener { annotation ->
+            decorations.findInputCallback<CircleNode, Circle, Unit>(
+                nodeMatchPredicate = { it.circle.id == annotation?.id && it.circleManager.layerId == circleManager.layerId },
+                nodeInputCallback = { onTap }
```
Comment (ianthetechie) [inline compose/src/main/java/com/maplibre/compose/ramani/MapLibreCompose.kt:183]: Can we just do `nodeInputCallback = onTap`? (Same idea on L191, 265, and 273).

Comment (Archdoog) [reply compose/src/main/java/com/maplibre/compose/ramani/MapLibreCompose.kt:183]: Unfortunately not, the brackets correctly capture the value which is associated with the `CircleNode`, `SymbolNode`, etc.

File: compose/src/main/java/com/maplibre/compose/ramani/MapLibreCallbacks.kt (hunk 1 above)
Comment (Archdoog) [reply compose/src/main/java/com/maplibre/compose/ramani/MapLibreCallbacks.kt]: What we see here, is a function that I was going to originally transform within. But didn't need to in the end 🤣😅

File: compose/src/main/java/com/maplibre/compose/ramani/MapLibreCompose.kt (hunk 2 above)
Comment (ianthetechie) [reply compose/src/main/java/com/maplibre/compose/ramani/MapLibreCompose.kt:183]: Oh, fun....

**Pull request merged**